### PR TITLE
jade: pass any explicit asset/value proofs in commitment data

### PR DESCRIPTION
**Pull request by JamieDriver**
Pass any `asset_blind_proof` if present in jade commitment data - same for `value_blind_proof`.

### CODE REVIEW

File: green_cli/authenticators/jade.py [hunk 1]
```diff
@@ -348,14 +348,20 @@ def _map_input(input: Dict) -> Dict:
             }
 
             # If non-trivial tx, also send any blinding information per input
-            if txtype != tx.TxType.SEND_PAYMENT and 'assetblinder' in input:
-                mapped['abf'] = bytes.fromhex(input['assetblinder'])[::-1]
+            if txtype != tx.TxType.SEND_PAYMENT and input.get('is_blinded'):
                 mapped['asset_id'] = bytes.fromhex(input['asset_id'])
                 mapped['asset_generator'] = bytes.fromhex(input['asset_tag'])
+
+                if 'assetblinder' in input:
```
Comment (jgriffiths) [inline green_cli/authenticators/jade.py:355]: Its possible albeit potentially inefficient to provide both the blinder and the blind proofs. If the proofs are present they should be verified IMO even if the blinder is provided.

Comment (jgriffiths) [reply green_cli/authenticators/jade.py:355]: Note this kind of thing can happen e.g. when callers augment their own PSETs for signing then strip out the sigs to poke into a PSET they are sending someone else.

**Comment by jgriffiths** [comment]
utack 4e9af9e7f585e71d977d8fa9e1f377f1eb9a727d

**Comment by JamieDriver** [comment]
Thanks mate.

**Pull request merged**